frontend/catalyst/jax_primitives.py

```diff
@@ -105,6 +113,14 @@ def __init__(self):
 class AbstractQreg(AbstractValue):
     """Abstract quantum register."""
 
+    hash_value = hash("AbstractQreg")
```

Comment:
Should the descendant class below overwrite this value?

Reply:
I don't think it matters, we don't really use them.